src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -6,8 +6,41 @@ import { stateRequiredCountries } from '../../common/schemaform/definitions/addr
 import { transformForSubmit } from '../../common/schemaform/helpers';
 
 
+const siblings = ['treatments', 'privateRecordReleases', 'privateRecords', 'additionalDocuments'];
+
+export function flatten(data) {
+  const formData = Object.assign({}, data);
```

Comment:
This actually only does a shallow clone, so the references to all properties of `data` of type `Object` (including arrays because JavaScript) will be the same. This means when you modify `formData.disabilities`, it's also modifying `data.disabilities`, making this function "impure".

Not sure how much it _really_ matters, but if you want to make it pure, you can use the deep clone from `src/js/common/utils/data-utils/cloneDeep.js` (to avoid lodash dependency :wink: ). Alternatively, `set` and `omit` from `data-utils` will probably do the job as well, but...I don't know whether that's a better approach than just deep cloning the thing.

Reply:
Good point, will use reduce to build this instead.

Reply:
Good idea, will use the common data-utils instead.